gateway/ln-gateway/src/envs.rs

```diff
@@ -1,50 +1,77 @@
+/// Environment variable that specifies that Bitcoin network that the gateway
+/// should use. Must match the network of the Lightning node.
 pub const FM_GATEWAY_NETWORK_ENV: &str = "FM_GATEWAY_NETWORK";
 
-// Env variable to TODO
+/// Environment variable that specifies the default routing fees the gateway
+/// takes for outgoing and incoming payments. Only applied to newly joined
+/// federations.
 pub const FM_DEFAULT_GATEWAY_FEES_ENV: &str = "FM_DEFAULT_GATEWAY_FEES";
 
-// Env variable to TODO
+/// Environment variable that instructs the gateway how many route hints to
+/// include in LNv1 invoices.
 pub const FM_NUMBER_OF_ROUTE_HINTS_ENV: &str = "FM_NUMBER_OF_ROUTE_HINTS";
 
-// Env variable to TODO
+/// Environment variable that specifies the URL to connect to LND. Necessary for
+/// LND configuration.
 pub const FM_LND_RPC_ADDR_ENV: &str = "FM_LND_RPC_ADDR";
 
-// Env variable to TODO
+/// Environment variable that specifies the location of LND's TLS certificate.
+/// Necessary for LND configuration.
 pub const FM_LND_TLS_CERT_ENV: &str = "FM_LND_TLS_CERT";
 
-// Env variable to TODO
+/// Environment variable that specifies the location of LND's macaroon.
+/// Necessary for LND configuration.
 pub const FM_LND_MACAROON_ENV: &str = "FM_LND_MACAROON";
 
-// Env variable to TODO
-pub const FM_GATEWAY_LIGHTNING_ADDR_ENV: &str = "FM_GATEWAY_LIGHTNING_ADDR";
-
-// Env variable to TODO
+/// Environment variable that specifies the URL of an Esplora server. Necessary
+/// for LDK configuration.
 pub const FM_LDK_ESPLORA_SERVER_URL: &str = "FM_LDK_ESPLORA_SERVER_URL";
 
-// Env variable to TODO
+/// Environment variable that specifies the Bitcoin network that the LDK Node
+/// should use. Must match `FM_GATEWAY_NETWORK`. Necessary for LDK
+/// configuration.
 pub const FM_LDK_NETWORK: &str = "FM_LDK_NETWORK";
 
-// Env variable to TODO
+/// Environment variable the specifies the port that the LDK Node should use.
+/// Necessary for LDK configuration.
 pub const FM_PORT_LDK: &str = "FM_PORT_LDK";
 
+/// Environment variable that specifies the address that the Core Lightning
+/// extension should listen on. Will be removed in v0.6
+pub const FM_CLN_EXTENSION_LISTEN_ADDRESS_ENV: &str = "FM_CLN_EXTENSION_LISTEN_ADDRESS";
+
+/// Environment variable that specifies the URL that the gateway can use to
+/// connect to the Core Lightning extension. Will be removed in v0.6.
+pub const FM_GATEWAY_LIGHTNING_ADDR_ENV: &str = "FM_GATEWAY_LIGHTNING_ADDR";
+
+/// Environment variable that specifies the mnemonic that the gateway should use
+/// for ecash and the LDK Node should use for onchain funds. If not set, a
+/// mnemonic will be generated. This environment variable can be used for
+/// recovering from an existing mnemonic.
 pub const FM_GATEWAY_MNEMONIC_ENV: &str = "FM_GATEWAY_MNEMONIC";
 
+/// Environment variable that specifies the "module mode" the gateway should run
+/// in. Options are "LNv1", "LNv2", or "All". It is not recommended to run "all"
```

Comment:
[nit] Is "All" case sensitive? If so, then it may be more clear to `s/all/All`

Reply:
not case sensitive but good to be consistent. Fixed in the rebase :)